client/apps/articles_list/components/articles_list.jsx

```diff
@@ -0,0 +1,147 @@
+
+export class ArticlesList extends Component {
+  static propTypes = {
+    actions: PropTypes.object,
+    articles: PropTypes.array,
+    published: PropTypes.bool,
+    channel: PropTypes.object
+  }
+
+  state = {
+    articles: this.props.articles || [],
+    published: this.props.published,
+    offset: 0
+  }
+
+  componentDidMount () {
+    const { viewArticles } = this.props.actions
+    viewArticles()
+
+    const canLoadMore = debounce(this.canLoadMore, 300)
+    return $.onInfiniteScroll(canLoadMore)
+  }
+
+  canLoadMore = () => {
+    if ($('.filter-search__input').val()) { return }
+    $('.loading-spinner').fadeIn()
+    return this.fetchFeed(this.state.published, this.state.offset + 10, this.appendMore.bind(this))
+  }
+
+  setResults = (results) => {
+    return this.setState({articles: results})
+  }
+
+  setPublished (type) {
+    this.setState({published: type, offset: 0})
+    return this.fetchFeed(type, 0, this.setResults)
+  }
+
+  appendMore (results) {
+    const articles = this.state.articles.concat(results)
+    this.setState({articles})
+    return $('.loading-spinner').fadeOut()
+  }
+
+  fetchFeed (type, offset, cb) {
+    const feedQuery = query(`published: ${type}, offset: ${offset}, channel_id: "${sd.CURRENT_CHANNEL.id}"`)
+    return request
+      .post(sd.API_URL + '/graphql')
+      .set('X-Access-Token', sd.USER != null ? sd.USER.access_token : undefined)
+      .send({query: feedQuery})
+      .end((err, res) => {
+        if (err || !(res.body != null ? res.body.data : undefined)) { return }
+        this.setState({offset: this.state.offset + 10})
+        return cb(res.body.data.articles)
+      })
+  }
+
+  showEmptyMessage () {
+    return (
+      <div className='article=list__empty'>
+        <div>You haven’t written any articles yet.</div>
+        <div>Artsy Writer is a tool for writing stories about art on Artsy.</div>
+        <div>Get started by writing an article or reaching out to your liaison for help.</div>
+        <a
+          className='avant-garde-button avant-garde-button-black article-new-button'
+          href='/articles/new'
+          dangerouslySetInnerHTML={{ __html: $(icons()).filter('.new-article').html() + 'Write An Article' }}
+        />
+      </div>
+    )
+  }
+
+  showArticlesList () {
+    if (this.props.articles && this.props.articles.length) {
+      return (
+        <div className='articles-list__container'>
+          <div className='articles-list__title'>Latest Articles</div>
+          <FilterSearch
+            url={sd.API_URL + `/articles?published=${this.state.published}&channel_id=${sd.CURRENT_CHANNEL.id}&q=%QUERY`}
+            placeholder='Search Articles...'
+            collection={this.state.articles}
+            searchResults={this.setResults}
+            selected={null}
+            contentType='article'
+            checkable={false}
+            isArtsyChannel={this.props.channel.isArtsyChannel()}
+          />
```

Comment:
Since the store's channel is a plain object, we don't have the isArtsyChannel method here -- you can use `channel.type !== 'partner'` for the same result.